guides/source/active_record_migrations.md

```diff
@@ -46,112 +54,124 @@ class CreateProducts < ActiveRecord::Migration[8.0]
-Before this migration is run, there will be no table. After, the table will
-exist. Active Record knows how to reverse this migration as well: if we roll
-this migration back, it will remove the table.
-
-On databases that support transactions with statements that change the schema,
-each migration is wrapped in a transaction. If the database does not support this
-then when a migration fails the parts of it that succeeded will not be rolled
-back. You will have to rollback the changes that were made by hand.
-
-NOTE: There are certain queries that can't run inside a transaction. If your
-adapter supports DDL transactions you can use `disable_ddl_transaction!` to
-disable them for a single migration.
-
-### Making the Irreversible Possible
-
-If you wish for a migration to do something that Active Record doesn't know how
-to reverse, you can use `reversible`:
+This migration adds a table called `products` with a string column called `name`
+and a text column called `description`. A primary key column called `id` will
+also be added implicitly, as it's the default primary key for all Active Record
+models. The `timestamps` macro adds two columns, `created_at` and `updated_at`.
+These special columns are automatically managed by Active Record if they exist.
 
 ```ruby
-class ChangeProductsPrice < ActiveRecord::Migration[8.0]
-  def change
-    reversible do |direction|
-      change_table :products do |t|
-        direction.up   { t.change :price, :string }
-        direction.down { t.change :price, :integer }
-      end
-    end
+# db/schema.rb
+ActiveRecord::Schema[8.0].define(version: 2024_05_02_100843) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "products", force: :cascade do |t|
+    t.string "name"
+    t.text "description"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
   end
 end
 ```
 
-This migration will change the type of the `price` column to a string,
-or back to an integer when the migration is reverted. Notice the block being
-passed to `direction.up` and `direction.down` respectively.
+We define the change that we want to happen moving forward in time. Before this
+migration is run, there will be no table. After it is run, the table will exist.
+Active Record knows how to reverse this migration as well; if we roll this
+migration back, it will remove the table. You can read more about rolling back
+migrations [here](#rolling-back).
+
+After defining the change that we want to occur moving forward in time, it's
+essential to consider the reversibility of the migration. While Active Record
+can manage the forward progression of the migration, ensuring the creation of
+the table, the concept of reversibility becomes crucial. With reversible
+migrations, not only does the migration create the table when applied, but it
+also enables smooth rollback functionality. In case of reverting the migration,
```

Comment:
```suggestion
also enables smooth rollback functionality. In case of reverting the migration mentioned in the above example,
```

Reply:
I was referring to the migration at [line number 65](https://github.com/rails/rails/pull/51928/files/a9cb47b8db7efa0eae2c1c601412cadbf6a1b274#diff-2c180fd99974672183e98de5800facb91ae57e633415a23b00cceefbab0b366aR65). I think the doc should clearly emphasise what type of migration can be reverted. Thats why i suggested to add `mentioned in the above example`.